src/HCubature.jl

```diff
@@ -121,14 +121,14 @@ function hcubature_(f, a::SVector{n,T}, b::SVector{n,T}, norm, rtol_, atol, maxe
     return I,E
 end
 
-function hcubature_(f, a::AbstractVector{T}, b::AbstractVector{S},
+function hcubature(f, a::AbstractVector{T}, b::AbstractVector{S},
```

Comment:
We need access to the internal function which operates on StaticArrays to make things type stable. https://github.com/JuliaMath/HCubature.jl/pull/43/files#diff-2fcc0f866bfa647873cdff9e4037fe8c2a1f1f8ec052573a4ae777cd329e60faR49

Also, I don't think making the distinction between internal and external helps in anway.

Reply:
Actually, keeping the internal functions works too. But still curious how internal functions help?

Reply:
Since [tests here](https://github.com/SciML/Integrals.jl/pull/89) pass, I guess [this function call](https://github.com/JuliaMath/HCubature.jl/blob/master/src/HCubature.jl#L177) is sufficient/